doc/Obtaining.rst

```diff
@@ -88,43 +88,12 @@ and other HACL* users have also developed bindings for other programming languag
 OCaml
 ^^^^^
 
-The KaRaMeL compiler auto-generates ``ocaml-ctypes`` bindings for HACL*. On top
-of these "raw" bindings, we add a high-level wrapper that uses functors, shares
-type signatures, performs run-time checks and offers a much more idiomatic API.
-
-They are both available as opam packages and can be installed using
-``opam install hacl-star`` or, to get the low-level bindings only,
-``opam install hacl-star-raw``.
-
-From source, the low-level bindings can be installed by running
-``make install-hacl-star-raw`` in ``dist/gcc-compatible``. Then, the high-level
-wrapper can be installed by running ``dune build && dune install`` in
-``bindings/ocaml``.
-
-Documentation for the high-level API is available `here
-<https://hacl-star.github.io/ocaml_doc/>`_.
+OCaml bindings were moved to the `cryspen/hacl-packages` repository.
```

Comment:
There's https://tech.cryspen.com/hacl-packages/hacl-js/index.html and https://tech.cryspen.com/hacl-packages/hacl-ocaml/index.html.
Can you add documentation that you think makes sense there?

Reply:
I think the links you provided should be the entry points to the documentation, and we should add anything relevant there. Leaving anything in the hacl-star repository would require maintenance.

Reply:
Yeah I agree. I just thought there might be interesting information here that you remove and that you could add to the two linked pages.